src/server/pfs/server/api_server.go

```diff
@@ -69,14 +69,14 @@ func (a *apiServer) InspectRepo(ctx context.Context, request *pfs.InspectRepoReq
 	func() { a.Log(request, nil, nil, 0) }()
 	defer func(start time.Time) { a.Log(request, response, retErr, time.Since(start)) }(time.Now())
 
-	return a.driver.inspectRepo(ctx, request.Repo, request.IncludeAuth)
+	return a.driver.inspectRepo(ctx, request.Repo, true)
 }
 
 func (a *apiServer) ListRepo(ctx context.Context, request *pfs.ListRepoRequest) (response *pfs.ListRepoResponse, retErr error) {
 	func() { a.Log(request, nil, nil, 0) }()
 	defer func(start time.Time) { a.Log(request, response, retErr, time.Since(start)) }(time.Now())
 
-	repoInfos, err := a.driver.listRepo(ctx, request.Provenance, request.IncludeAuth)
+	repoInfos, err := a.driver.listRepo(ctx, request.Provenance, true)
```

Comment:
Should we just update `driver.listRepo` if `includeAuth` is never gonna be `false`?

Reply:
There are a few places where it's still false, notably createRepo (https://github.com/pachyderm/pachyderm/blob/9aa9641f1d4279ed6664e54874463e4f37d5b739/src/server/pfs/server/driver.go#L310) and deleteAll (https://github.com/pachyderm/pachyderm/blob/9aa9641f1d4279ed6664e54874463e4f37d5b739/src/server/pfs/server/driver.go#L1902)